### PR TITLE
Integrates Japanese Language

### PR DESCRIPTION
**Pull request by ray-hrst**
- Integrates Japanese language into the skill
- Extracts English and Japanese "tips" into strings.xml for localization purposes
- Rephrases and cleans up README

### CODE REVIEW

File: app/src/main/java/com/robotemi/welcomingbtob/tips/TipsViewImpl.kt [hunk 1]
```diff
@@ -57,15 +57,6 @@ class TipsViewImpl : RelativeLayout {
...
         when (robot.wakeupWord.toLowerCase(Locale.ENGLISH)) {
             Constants.WAKEUP_WORD_ALEXA,
-            Constants.WAKEUP_WORD_ALEXA_JAPAN -> {
-                tips.clear()
-                tips.add(String.format("Try “Alexa, tell my temi to go to %s”", locationForTips))
-                tips.add("Try “Alexa, tell my temi to follow me”")
-                tips.add(String.format("Try “Alexa, tell my temi to call %s”", callForTips))
-                tips.add("Try “Alexa, tell my temi to take a selfie”")
-                tips.add("Try “Alexa, tell my temi to take a video”")
-                tips.add("Try “Alexa, tell my temi to take a GIF”")
```
Comment (lcgao) [inline app/src/main/java/com/robotemi/welcomingbtob/tips/TipsViewImpl.kt:67]: Why delet all the tips related Alexa even this is not only for Alexa Japan?

File: build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ buildscript {
 
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
```
Comment (lcgao) [inline build.gradle:16]: Please explain for this update.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Tue Sep 24 16:44:14 CST 2019
+#Wed May 27 17:47:52 JST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```
Comment (lcgao) [inline gradle/wrapper/gradle-wrapper.properties:6]: Please explain for this update.

File: app/src/main/java/com/robotemi/welcomingbtob/tips/TipsViewImpl.kt (hunk 1 above)
Comment (ray-hrst) [reply app/src/main/java/com/robotemi/welcomingbtob/tips/TipsViewImpl.kt:67]: You're right. I meant to remove it only for Japan. I'll make the appropriate changes.

File: build.gradle (hunk 2 above)
Comment (ray-hrst) [reply build.gradle:16]: Gradle 3.6.0 fixes various known issues 3.5.3, e.g. linting is slow, missing manifest class, etc. Is there a strong reason not to upgrade this dependency?

**Comment by ray-hrst** [comment]
@lcgao I fixed the Alexa-English issue. Please double-check.
As for the Gradle version, I would suggest using a more recent version because it fixes some features that you may or may not use, but that's entirely up to you.

File: build.gradle (hunk 2 above)
Comment (lcgao) [reply build.gradle:16]: Okay, great.

**Comment by lcgao** [comment]
> @lcgao I fixed the Alexa-English issue. Please double-check.
> As for the Gradle version, I would suggest using a more recent version because it fixes some features that you may or may not use, but that's entirely up to you.

Merged.

**Pull request merged**